### ISSUE TITLE
Can not start app [F5] ScErrDebugSequenceFailUnexpect

### ISSUE THREAD

**Issue by andwah** [Starcounter/Starcounter#3426]
With the latest code 2016-03-04 im getting problems starting apps.

1. Pull latest `level1 `from `develop `code and build.
2. Create new Starcounter project.
3. F5

VIsual Studio Output:
```
Severity	Code	Description	Project	File	Line
Error		The debugging sequence failed with an unexpected error. Consult the Visual Studio activity log for additional information. Error summary: Object reference not set to an instance of an object. (ScErrDebugSequenceFailUnexpect (SCERR12016))
```
Worked just fine a day ago.

**Comment by per-samuelsson** [comment]
And nothing in the logs?

**Comment by alemoi** [comment]
Anything more about exception in output? Worrying that could be caused by my changes in Response class. Does manual start using star works for you?

**Comment by andwah** [comment]
@per-samuelsson Nothing in log :-(

**Comment by andwah** [comment]
I can start the app with `Star` but i can not start it within Visual studio [F5] (debugging)

**Comment by andwah** [comment]
I will try to reinstall the Starcounter extention..

**Comment by andwah** [comment]
Update: Reinstalling the extension did not solve my problem

**Comment by per-samuelsson** [comment]
FWIW, @eriohl reported the same thing earlier on Gitter.

**Comment by per-samuelsson** [comment]
I'll try bringing things down on my machine and try it here to see if I get the same. If so, I can debug it for you.

**Comment by alemoi** [comment]
Per do you remember it was somehow possible to debug extension itself, like attaching VS process in another process? I just need the stack trace to understand where it occurs.

**Comment by per-samuelsson** [comment]
> Per do you remember it was somehow possible to debug extension itself, like attaching VS process in another process?

Yes, of course it is.

Just fire up one instance of VS where the app project is, e.g. "devenv.exe" PID: 123. The open another instance (normally, just the one where Level1 is loaded to have all the code at hand). In this instance, "Attatch to process ..." and attach it to "devenv.exe" PID 123.

**Comment by per-samuelsson** [comment]
Another way is to just specify `--sc-debug` in "Command-line arguments" in any Starcounter project loaded into a VS instance, and it will prompt you to attach the debugger to the hosting VS instance just at the start of the extension debugging sequence.

**Comment by per-samuelsson** [comment]
Like:

1. Open "MyApp.exe" in VS
2. Go to `Properties | Debug` and put `--sc-debug` in command-line parmeters.
3. Hit F5.

You'll be prompted to attach.

**Comment by per-samuelsson** [comment]
So you are on this one then @alemoi? Then I won't bother until you tell me to, ok?

**Comment by alemoi** [comment]
Yeah, I will check this one!

**Comment by alemoi** [comment]
I found the issue as I expected to be - just a missed initialization. Sorry, my bad. The fix 5de08c3fd9b33c0154af910681375982ce93ebd6 is pushed to develop and pnext. @per-samuelsson thx for the help in how to debug vs extension! @andwah thx for reporting :)

**Comment by andwah** [comment]
@alemoi Thanks for the quick response.. i have pulled the latest and all works again as expected.
